tools/ci/stage.sh

```diff
@@ -26,7 +26,7 @@ STAGE_CONNECTORS_2="connect_2"
 STAGE_TESTS="tests"
 STAGE_MISC="misc"
 STAGE_CLEANUP="cleanup"
-STAGE_FINEGRAINED_RESOURCE_MANAGEMENT="finegrained_resource_management"
+STAGE_DECLARATIVE_RESOURCE_MANAGEMENT="declarative_resource_management"
```

Comment:
I wonder if we can simply remove this stage. I'd expect there to be barely any changes to the declarative slot manager before it gets removed. Maybe relying on the unit tests would be good enough.

Reply:
Either way, we would need a JIRA ticket to remind us to remove declarative slot manager in the next release. A ticket with Blocker priority and 1.19/2.0 fixVersion should do the job.

Reply:
Ticket are created, https://issues.apache.org/jira/browse/FLINK-31449